### DISCUSSION TITLE
Info older Gens transferring to Gen 9?

### DISCUSSION THREAD

**Discussion by DarylCAH** [kwsch/PKHeX#4160] [Q&A] [answered]
Is there a list anywhere of what Gens are currently legally transferrable to newer Gens?

My plan is to play through all the previous Gens and move as many mons up the Gens ending with them in S/V as I can.

I'll be playing all the handheld Gens on my cfw 3DS and any Switch titles on cfw Switch.

I want to do all the transfers through pkhex, avoiding Bank and Home.

Thanks for any info

**Comment by ReignOfComputer** [comment, marked as answer]
Just Google any "Pokemon Home Transference Chart". Keep in mind transferring through PKHeX means they will not have a Home Tracker and thus, not moveable using official platforms in the future.

**Reply by DarylCAH** [reply]
Thanks, found an image on Reddit that makes sense.
I have no intention of ever using them in online battles/transfers.
